build/.moduleignore.darwin

```diff
@@ -0,0 +1,2 @@
+@vscode/windows-mutex/index.js
+@vscode/windows-mutex/**/*.node
```

Comment:
what about `*.md`, `src/*.cc|h`, `*.json`, `*.gyp` ?

Reply:
I see `.gyp` and `src` are covered here https://github.com/microsoft/vscode/blob/70daab796e44df766e652ccb6ba9b6997aeb46e4/build/.moduleignore#LL34C29-L34C29